lua/neotest/config/init.lua

```diff
@@ -26,6 +26,7 @@ vim.cmd([[
 ---@field icons table<string, string>
 ---@field highlights table<string, string>
 ---@field floating neotest.Config.floating
+---@field default_strategy string
```

Comment:
This can also be a function that returns a `neotest.Process` as well. I haven't added the process type to the doc file yet so I'd be happy just saying it can be `string | function` and let users discover themselves since they'll need to be adventurous to implement :sweat_smile: